packages/nx/src/utils/globs.ts

```diff
@@ -2,3 +2,14 @@ export function combineGlobPatterns(...patterns: (string | string[])[]) {
   const p = patterns.flat();
   return p.length > 1 ? '{' + p.join(',') + '}' : p.length === 1 ? p[0] : '';
 }
+
+export const GLOB_CHARACTERS = new Set(['*', '|', '{', '}', '(', ')', '[']);
```

Comment:
These are all the characters I found when doing the whole glob conversion stuff in the Rust side. Are they needed here as well (like the `?`, `@`, etc)?

https://github.com/nrwl/nx/blob/0b0db782f7b121b71d52419db675b90fa3080010/packages/nx/src/native/glob.rs#L110-L124

Reply:
I don't think so... ?, @, +, ! are only valid in combination with paraenthises